### PR TITLE
admin/checking-dep-upper-bounds

### LINKED ISSUE #14: Support for newer `xarray` versions?
Hi, we recently migrated from `aicsimageio` to `bioio` at https://github.com/multiview-stitcher/multiview-stitcher and realised `bioio-czi` has a requirement for

https://github.com/bioio-devs/bioio-czi/blob/5a26a300a128a2ac0e3b7d605432789ca427f553/pyproject.toml#L37

Could this be updated to a newer version of `xarray`?

Thanks a lot in advance and I'm happy to help with any testing.

### PR DESCRIPTION
**Pull request by evamaxfield**
### Link to Relevant Issue

This pull request resolves #14 

### Description of Changes

Using CI to check if upper bounds on deps can be removed / what the issues are

### CODE REVIEW

**Review by BrianWhitneyAI — APPROVED** [review]
Hm dont know why this was pinned. LGTM!

**Comment by evamaxfield** [comment]
Going to try and remove other upper bounds too (dask + numpy)

**Comment by evamaxfield** [comment]
@BrianWhitneyAI seems like all of these upper bounds can be removed. Maybe we should go through and do this to other plugins?

**Review by toloudis — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -29,12 +29,12 @@ dynamic = ["version"]
 dependencies = [
   "aicspylibczi>=3.1.1",
```
Comment (toloudis) [inline pyproject.toml:30]: might as well....assuming the tests pass
```suggestion
  "aicspylibczi>=3.2.0",
```

**Comment by evamaxfield** [comment]
Everything is 🟢 🎉 ! @toloudis do I have your permission to release a new version after I merge this and let main branch CI run?

**Comment by toloudis** [comment]
> Everything is 🟢 🎉 ! @toloudis do I have your permission to release a new version after I merge this and let main branch CI run?

Yup!

**Pull request merged**